homeassistant/components/hvv_departures/sensor.py

```diff
@@ -0,0 +1,215 @@
+"""Sensor platform for hvv."""
+from datetime import datetime, timedelta
+import logging
+
+from aiohttp import ClientConnectorError
+from pygti.exceptions import InvalidAuth
+from pygti.gti import GTI, Auth
+
+from homeassistant.const import DEVICE_CLASS_TIMESTAMP
+from homeassistant.helpers import aiohttp_client
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+
+from .const import ATTRIBUTION, DOMAIN, MANUFACTURER
+
+MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=1)
+MAX_LIST = 20
+MAX_TIME_OFFSET = 360
+ICON = "mdi:bus"
+UNIT_OF_MEASUREMENT = "min"
+
+ATTR_ATTRIBUTION = "attribution"
```

Comment:
Import this constant from const.py.

Reply:
All of these are only used on the sensor platform. There is another binary_sensor platform planned, but they use a different icon and not the same attributes. See the developer documentation: https://developers.home-assistant.io/docs/en/next/creating_platform_code_review.html#0-common

As these constants are not widely used but just at platform level, they should be defined in sensor.py

Reply:
Sorry I was actually just referring to only `ATTR_ATTRIBUTION`.